spec.md

```diff
@@ -0,0 +1,26 @@
+# Your spec's journey
+* Create a new wiki page using the spec template below, to jot your thoughts down.
+* Create an [issue](https://github.com/NuGet/Home/issues) to track the feature or link an existing issue to the new wiki page. Engage the community on the feature. Feel free to tweet it from the @NuGet handle or ask the PM to tweet it out.
```

Comment:
This repo is not in the NuGet org, so I don't think it's appropriate to direct customers to file issues in NuGet/Home.